src/app/components/layout/Browse.tsx

```diff
@@ -49,11 +49,11 @@ export const Browse = () => {
         <FilterModal closeHandler={() => setIsModalVisible(false)} />
       )}
       <ul className="usa-card-group">
-        {institutionsArray?.map((school: College) => (
+        {filteredInstitutions?.map((school: College) => (
```

Comment:
do we want a "no results found" display?

Reply:
Yep we need one.